modules/troubleshooting-network-observability-loki-resource-exhausted.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/troubleshooting-network-observability.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-troubleshooting-loki-resource-exhausted_{context}"]
+= Troubleshooting Loki ResourceExhausted error
+Loki may return a "ResourceExhausted" error when network flow data sent by Network Observability exceeds the configured maximum message size. When using the Loki operator, this maximum message size is configured to 100 MiB. 
+
+.Procedure
+. Navigate to *Operators* -> *Installed Operators*, viewing *All projects* from the *Project* dropdown.
+. Look under the *Provided APIs* heading for the Netwok Observability Operator, select *Flow Collector* the the *YAML view* tab. 
```

Comment:
Typo: Netwok -> Network

Reply:
also, `select *Flow Collector* the the *YAML view* tab. ` should be `select *Flow Collector* then the *YAML view* tab. `

Reply:
Thank you!